pwiz_tools/Skyline/SettingsUI/FullScanSettingsControl.cs

```diff
@@ -953,7 +953,8 @@ public void ModifyOptionsForImportPeptideSearchWizard(ImportPeptideSearchDlg.Wor
 
                 ProductMassAnalyzer = PrecursorMassAnalyzer;
 
-                if (workflow == ImportPeptideSearchDlg.Workflow.dia && Settings.Default.IsolationSchemeList.Count > 1)
+                if (workflow == ImportPeptideSearchDlg.Workflow.dia && Settings.Default.IsolationSchemeList.Count > 1 &&
+                    settings.TransitionSettings.FullScan.IsolationScheme == null)
                 {
                     comboIsolationScheme.SelectedIndex = 1;

```

Comment:
Also, you should add a comment here to clarify what this is doing. i.e. What does 1 mean?